src/main/java/bio/terra/service/filedata/google/firestore/FireStoreUtils.java

```diff
@@ -36,21 +38,11 @@
     private final Logger logger = LoggerFactory.getLogger(FireStoreUtils.class);
     private static final int SLEEP_BASE_SECONDS = 1;
 
-    private ConfigurationService configurationService;
+    private int FIRESTORE_RETRIES;
 
     @Autowired
     public FireStoreUtils(ConfigurationService configurationService) {
-        this.configurationService = configurationService;
-    }
-
-    public int getFirestoreRetries() {
-        try {
-            return configurationService.getParameterValue(ConfigEnum.FIRESTORE_RETRIES);
-        } catch (Exception ex) {
-            logger.info("No value set for FIRESTORE_RETRIES parameter. Defaulting to 1.");
-            return 1;
-        }
-
+        this.FIRESTORE_RETRIES = configurationService.getParameterValue(ConfigEnum.FIRESTORE_RETRIES);
```

Comment:
This is now only used in one place, in the FirestoreUtils, and it should be able to handle a retry value of 0.